lib/options-manager.js

```diff
@@ -66,26 +66,50 @@ const DEFAULT_CONFIG = {
  * With `engines.node` set to `>=8` the rule `plugin/rule` will be used with the config `{prop: 'node-8-conf'}`.
  */
 const ENGINE_RULES = {
-	'node/prefer-global/text-decoder': {
-		'11.0.0': [
-			'error',
-			'always'
-		]
+	'unicorn/prefer-spread': {
+		'5.0.0': 'off'
+	},
+	'unicorn/no-new-buffer': {
+		'5.10.0': 'off'
+	},
+	'prefer-rest-params': {
+		'6.0.0': 'off'
+	},
+	'prefer-destructuring': {
+		'6.0.0': 'off'
+	},
+	'promise/prefer-await-to-then': {
+		'7.6.0': 'off'
+	},
+	'prefer-object-spread': {
+		'8.3.0': 'off'
+	},
+	'node/prefer-global/url-search-params': {
+		'10.0.0': 'off'
+	},
+	'node/prefer-global/url': {
+		'10.0.0': 'off'
+	},
+	'no-useless-catch': {
+		'10.0.0': 'off'
+	},
+	'prefer-named-capture-group': {
+		'10.0.0': 'off'
```

Comment:
I noticed now that this no longer reports when I run XO on https://github.com/avajs/ava (It does with XO 0.26.0)

Its `node` version target is: `"node": ">=10.18.0 <11 || >=12.14.0 <13 || >=13.5.0"`

Reply:
I just tested that with `>=10.18.0 <11 || >=12.14.0 <13 || >=13.5.0` the current code turn off only the following rules:
  - `node/prefer-global/text-encoder`
  - `node/prefer-global/text-decoder`
  - `unicorn/prefer-flat-map`
  - `node/prefer-promises/dns`
  - `node/prefer-promises/fs`

Maybe you have an outdated version of `eslint-config-xo` so the rule is not enabled in the first place?

Reply:
I also tried to run xo 0.26.0 on the current codebase of https://github.com/avajs/ava and the rule doesn't report.

Reply:
Hmm, weird. I couldn't reproduce it multiple times earlier. Just tried reinstalling node_modules everywhere and now it of course works... Sorry for wasting your time.

Reply:
No problem! I'd rather spend more time to make sure it works.
Sometimes weird stuff happens with package-lock.json......